### PR TITLE
feat(kubernetes): remove kubernetes.jobs.append-suffix flag and logic…

### PR DESCRIPTION
**Pull request by dbyron0**
… to append a suffix to jobs

This is a follow-up to
https://spinnaker.io/community/releases/versions/1-22-0-changelog#breaking-change-suffix-no-longer-added-to-jobs-created-by-kuber
and simplifies the code enough to facilitate documentation of the behavior at
https://spinnaker.io/guides/user/kubernetes-v2/run-job-manifest/#cleaning-up-old-jobs.

Documentation PR [here](https://github.com/spinnaker/spinnaker.github.io/pull/2155)

### CODE REVIEW

File: clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java [hunk 1]
```diff
@@ -74,50 +72,28 @@ void setTask() {
...
-    // We're asserting that at least 4 characters are added after the hyphen; rather than
-    // overspecify the test by looking up exactly how many are added, we're just making sure that a
-    // reasonable number to guarantee randomness are added.
-    assertThat(suffix.length()).isGreaterThanOrEqualTo(5);
-  }
-
-  @ParameterizedTest
-  @ValueSource(booleans = {true, false})
-  void deploysJobWithGenerateName(boolean appendSuffix) {
+  void deploysJobWithGenerateName() {
```
Comment (german-muzquiz) [inline clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java:82]: I don't think this test is going to run without the `@Test` annotation

File: clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java [hunk 2]
```diff
@@ -74,50 +72,28 @@ void setTask() {
...
 
     assertThat(result.getManifestNamesByNamespace()).containsOnlyKeys(NAMESPACE);
     assertThat(result.getManifestNamesByNamespace().get(NAMESPACE))
         .containsExactlyInAnyOrder(DEPLOYED_JOB + GENERATE_SUFFIX);
   }
 
-  @ParameterizedTest
-  @ValueSource(booleans = {true, false})
-  void overridesNamespace(boolean appendSuffix) {
+  void overridesNamespace() {
```
Comment (german-muzquiz) [inline clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java:92]: Same comment as above

File: clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java (hunk 1 above)
Comment (dbyron0) [reply clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java:83]: Right you are: https://github.com/spinnaker/clouddriver/pull/5310/commits/11177f72c5350cd10616473e3d2f2db6782f2b1c

File: clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java (hunk 2 above)
Comment (dbyron0) [reply clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubernetesRunJobOperationTest.java:94]: https://github.com/spinnaker/clouddriver/pull/5310/commits/11177f72c5350cd10616473e3d2f2db6782f2b1c

**Review by german-muzquiz — APPROVED** [review]

**Comment by dbyron0** [comment]
Thanks @german-muzquiz !  May I ask for even more of your time to take a look at the release notes + doc change [here](https://github.com/spinnaker/spinnaker.github.io/pull/2155) ?

**Pull request merged**